influxql/parser_test.go

```diff
@@ -338,7 +337,7 @@ func TestParser_ParseStatement(t *testing.T) {
 
 		// CREATE USER statement
 		{
-			s: `CREATE USER testuser WITH PASSWORD pwd1337`,
+			s: `CREATE USER testuser WITH PASSWORD 'pwd1337'`,
```

Comment:
Surely the SQL example we should compare against is `CREATE USER` which doesn't require wrapping the password in call?

http://dev.mysql.com/doc/refman/5.0/en/create-user.html

However, my experiments show that MySQL also requires the password to be quoted (something I didn't realise) so we're as good as them. :-)